flair/embeddings/transformer.py

```diff
@@ -624,6 +627,10 @@ def __expand_sentence_with_context(self, sentence) -> Tuple[List[Token], int]:
             left_context = sentence.left_context(self.context_length, self.respect_document_boundaries)
             right_context = sentence.right_context(self.context_length, self.respect_document_boundaries)
 
+            if self.use_context_separator:
+                left_context = left_context + [Token("[KONTEXT]")]
+                right_context = [Token("[KONTEXT]")] + right_context
```

Comment:
Can we call this token `[CONTEXT]` instead of `[KONTEXT]`? I think we benefit from not mixing up two languages.
From my understanding, changing the text of the context won't change the training, as it is added as a special token.

Besides that, I would prefer that text to be extracted to a constant (similar to https://github.com/flairNLP/flair/blob/master/flair/models/sequence_tagger_utils/crf.py#L5 with the `START_TOKEN`), so we won't have the case that typos in a single string can break the code in future.